docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "tas-item",
+  "description": "A question item for TAS",
+  "definitions": {
+
+    "tas-singleLine": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-singleLine",
+      "title": "tas-string",
+      "description": "A text question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["singleLine"]
+        },
+        "minSize": {
+          "description": "Min number of characters",
+          "type": "number",
+          "default": 1
+        },
+        "maxSize": {
+          "description": "Max number of characters",
+          "type": "number",
+          "default": 140
+        }
+      }
+    },
+
+    "tas-multiline": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-multiline",
+      "title": "tas-multiline",
+      "description": "A multiline text question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["multiline"]
+        },
+        "minSize": {
+          "description": "Min number of characters",
+          "type": "number",
+          "default": 1
+        },
+        "maxSize": {
+          "description": "Max number of characters",
+          "type": "number",
+          "default": 1000
+        }
+      }
+    },
+
+    "tas-number": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-number",
+      "title": "tas-number",
+      "description": "A number question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["number"]
+        },
+        "maximum": {
+          "description": "Max number size",
+          "type": "number"
+        },
+        "minimum": {
+          "description": "Min number size",
+          "type": "number"
+        }
+      }
+    },
+
+    "tas-bool": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-bool",
+      "title": "tas-bool",
+      "description": "A boolean question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["bool"]
+        },
+        "trueString": {
```

Comment:
This indicates what to display for each option. Sometimes you want the candidate to have the options "Yes" and "No", other times you might want "true" and "false", or you could want to show a check box that says "☑ I agree with all of the above".